src/main/proto/wfa/measurement/internal/kingdom/exchange.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.kingdom;
+
+import "google/type/date.proto";
+import "src/main/proto/wfa/measurement/internal/kingdom/exchange_details.proto";
+
+option java_package = "org.wfanet.measurement.internal.kingdom";
+option java_multiple_files = true;
+
+// Proto representation of the internal Exchange entity type.
+message Exchange {
+  fixed64 external_recurring_exchange_id = 1;
+  fixed64 external_exchange_id = 2;
+  google.type.Date date = 3;
```

Comment:
We don't want to keep the time as well?

Reply:
We have no known use cases that require granularity finer than on a per-day basis.